<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>912</height>
   </rect>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="appFrame">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frameSettings">
         <property name="font">
          <font>
           <family>Consolas</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayoutSettings">
            <property name="horizontalSpacing">
             <number>20</number>
            </property>
            <property name="verticalSpacing">
             <number>10</number>
            </property>
            <item row="1" column="2">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxST_MapScale">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="decimals">
               <number>0</number>
              </property>
              <property name="maximum">
               <double>1000000.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>10.000000000000000</double>
              </property>
              <property name="value">
               <double>300.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="checkBoxST_VerticalShoot">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Vertical shooting</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QSpinBox" name="spinBoxST_StartSpeed">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <spacer name="verticalSpacer_3">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="10" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="label_14">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Projectile mass, kg</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Grid size, m</string>
              </property>
             </widget>
            </item>
            <item row="10" column="2">
             <spacer name="horizontalSpacer_4">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="10" column="1">
             <spacer name="verticalSpacer">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_9">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Start speed, m/s</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QLabel" name="label_10">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Hint:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="3">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Settings</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="checkBoxST_ComplexCalculations">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Complex calculation</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1" colspan="2">
             <widget class="QLabel" name="label_12">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ctrl + RMB - Set Scale points
Alt + RMB - Set Target points
(double click at same 
point - reset points)

Ctrl + Z - Clear Scale points
Ctrl + X - Clear Target Points

Alt + R - Hide/Show Settings
Alt + E - Hide/Show programm
Alt + Q - Show template for 
	vertical shoot</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="label_15">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Projectile caliber, mm</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QSpinBox" name="spinBoxST_ProjectileCaliber">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>150</number>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QSpinBox" name="spinBoxST_ProjectileMass">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>500</number>
              </property>
              <property name="value">
               <number>18</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayoutCalculations">
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="2" column="1">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Distance: </string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <spacer name="horizontalSpacer">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Calculation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="labelCL_Distance">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>- m</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Point 2 (we):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelCL_Azimut">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="labelCL_Time">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QLabel" name="labelCL_Point2">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Azimuth:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <spacer name="verticalSpacer_2">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Meters per pixel:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Point 1 (target):</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="labelCL_Point1">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <spacer name="horizontalSpacer_6">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="4">
          <widget class="QLabel" name="labelCL_Scale">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_11">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Time, s</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Angle:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="labelCL_Angle">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>20</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
